packages/eslint-plugin-template/src/rules/no-inline-styles.ts

```diff
@@ -0,0 +1,129 @@
+import type {
+  ParseSourceSpan,
+  TmplAstElement,
+} from '@angular-eslint/bundled-angular-compiler';
+import {
+  createESLintRule,
+  getTemplateParserServices,
+} from '../utils/create-eslint-rule';
+
+type Options = [
+  {
+    readonly allowNgStyle?: boolean;
+    readonly allowBindToStyle?: boolean;
+  },
+];
+const DEFAULT_OPTIONS: Options[number] = {
+  allowNgStyle: false,
+  allowBindToStyle: false,
+};
+export type MessageIds = 'noInlineStyles';
+export const RULE_NAME = 'no-inline-styles';
+
+export default createESLintRule<Options, MessageIds>({
+  name: RULE_NAME,
+  meta: {
+    type: 'suggestion',
+    docs: {
+      description:
+        'Using html inline styles using style attribute is prohibited, use classes instead.',
```

Comment:
You have written this `description` as if it's a lint error, which is not what we want to do. The `description` is only used to describe why the rule exists, not give instructions/imply it is the "right" thing to do.

```suggestion
        'Disallows the use of inline styles in HTML templates',
```